src/control.c

```diff
@@ -470,18 +470,22 @@ boolean_t control_service(const char *S, Action_Type A) {
                 case Action_Restart:
                         LogInfo("'%s' trying to restart\n", s->name);
                         // Restart this service only if all children that depend on it were stopped
```

Comment:
This log line is now inaccurate.

I think there are some problems if dependent services fail to stop. Also, I think you should log if we fail to stop dependent services. Suggest:

```
if (_doDepend(s, Action_Unmonitor, false)) {
   if (_doDepend(s, Action_Stop, false))
   {
       // Continue with restart action
   }
   else
   {
      // Stop has failed, resume monitoring:
      LogError("Failed to stop dependent services while restarting '%s'\n", s->name);

      if (!_doDepend(s, Action_Monitor, false))
      {
           LogError("Failed to resume monitoring dependent services when failed to stop while restarting '%s'\n", s->name);
      }
   }
}
else
{
   LogError("Failed to stop monitoring dependent services while restarting '%s'\n", s->name);
}

I assume this change is to prevent monit complaining about dependent services not being in the correct state when a dependant service is being restarted?